libusb.go

```diff
@@ -235,6 +235,9 @@ func (libusbImpl) getDeviceDesc(d *libusbDevice) (*DeviceDesc, error) {
 		SubClass:             Class(desc.bDeviceSubClass),
 		Protocol:             Protocol(desc.bDeviceProtocol),
 		MaxControlPacketSize: int(desc.bMaxPacketSize0),
+		iSerialNumber:        int(desc.iSerialNumber),
+		iProduct:             int(desc.iProduct),
+		iManufacturer:        int(desc.iManufacturer),
```

Comment:
I'd also like to have all of that covered by unittests. This will require some changes to fakelibusb_*:

* fakelibusb_devices will need a set of predefined descriptors for the test devices. The fakedevices var holds a slice of []*DeviceDesc, we should change it to []struct{devDesc *DeviceDesc, strDesc map[int]string}.
* fakeLibusb.getDevices needs to be updated appropriately to return list of d.devDesc instead of d
* fakelibusb_test will need a different implementation of getStringDesc, so that it returns the descriptor from the test devices. Just iterate over fakeDevices, if devDesc == f.handles[d], then return strDesc[idx]. Or something like that.

Once these are in place, you can add a test to device_test.go, somewhere within TestClaimAndRelease. Somewhere around line 46. You can also add a test for config description (around line 55) and for interface description (66)